rust/private/clippy.bzl

```diff
@@ -146,6 +165,10 @@ rust_clippy_aspect = aspect(
     fragments = ["cpp"],
     host_fragments = ["cpp"],
     attrs = {
+        "_capture_output": attr.label(
+            doc = "If true, write clippy output as a build output, rather than printing it",
```

Comment:
Could you please also mention that the clippy output respects the configured `error_format`?

Reply:
Done